src/Polysemy/Trace.hs

```diff
@@ -82,7 +82,7 @@ runTraceList = runOutputList . reinterpret (
 --
 -- @since 1.0.0.0
 outputToTrace
-  :: Member Trace r
+  :: forall w r a. Member Trace r
```

Comment:
```suggestion
  :: forall w r a
   . Member Trace r
```

Reply:
Happy to merge after this. Thanks again, @juanpaucar

Reply:
Done :)